x-pack/plugins/apm/server/lib/rum_client/ui_filters/local_ui_filters/__snapshots__/index.test.ts.snap

```diff
@@ -44,11 +44,6 @@ Object {
               },
             },
           },
-          Object {
-            "term": Object {
-              "service.environment": "test",
-            },
-          },
```

Comment:
is this intentional?

Reply:
Yes. `esFilter` was a property of `setup` but now it's removed. The places where it was used in UX have been replaced with a call to `getEsFilter` with the uiFilters.